todo-app/src/todoDescription.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+class todoDescription extends React.Component {
```

Comment:
Component names should be named using [pascal case](http://wiki.c2.com/?PascalCase)! And same the file name!